README.md

```diff
@@ -11,7 +11,7 @@
 
 Phinx makes it ridiculously easy to manage the database migrations for your PHP app. In less than 5 minutes, you can install Phinx and create your first database migration. Phinx is just about migrations without all the bloat of a database ORM system or framework.
 
-**Check out [docs.phinx.org/en/latest/](http://docs.phinx.org/en/latest/) ([EN](http://docs.phinx.org/en/latest/), [ZH](https://tsy12321.gitbooks.io/phinx-doc/)) for the comprehensive documentation.**
+**Check out [book.cakephp.org/4/en/phinx.html](https://book.cakephp.org/4/en/phinx.html) ([EN](https://book.cakephp.org/4/en/phinx.html), [ZH](https://tsy12321.gitbooks.io/phinx-doc/)) for the comprehensive documentation.**
```

Comment:
this links content will remove soon
and will move to `https://book.cakephp.org/phinx/0` or for latest `https://book.cakephp.org/phinx`

Reply:
just use the  `https://book.cakephp.org/phinx` IMO
this link will fix soon

Reply:
you can use the `https://book.cakephp.org/phinx` now 
now it works well

Reply:
Done